kratos/python_scripts/restart_utility.py

```diff
@@ -175,6 +200,48 @@ def CreateOutputFolder(self):
                 os.makedirs(folder_path)
             self.model_part.GetCommunicator().GetDataCommunicator().Barrier()
 
+    def GetRestartFiles(self):
+        """Return a dictionary of stepID - restart_file_list dictionary that stores sets of restart files for each step."""
+        restart_path    = Path( self.__GetFolderPathSave() )
+        restart_files   = {}
+        if restart_path.is_dir():
+
+            file_name_data_collector = KratosMultiphysics.FileNameDataCollector( self.model_part, self._GetFileNamePattern(), {} )
+            file_name_data_list = file_name_data_collector.GetFileNameDataList()
+
+            for file_name_data in file_name_data_list:
+                # Get step id
+                if self.restart_control_type_is_time:
+                    step_id = file_name_data.GetTime()
+                else:
+                    step_id = file_name_data.GetStep()
+
+                # Check if this step has entries already,
+                # and add the file to the list of tracked files
+                if step_id in restart_files:
+                    restart_files[step_id].append( file_name_data.GetName() )
```

Comment:
Shouldn't this be 
```suggestion
                    restart_files[step_id].append( file_name_data.GetFileName() )
```

Reply:
That should have been fixed in [
d19fdca ](https://github.com/KratosMultiphysics/Kratos/pull/7987/commits/d19fdca39c189be085a116cdb153518f5f9bf5f9). I'll move from my fork to the main repo and check why that wasn't applied.